af/fastpath/debian/control

```diff
@@ -14,6 +14,7 @@ Depends: ${misc:Depends},
  ${python3:Depends},
  python3-paramiko,
  python3-statsd,
- python3-systemd
+ python3-systemd,
+ nginx
```

Comment:
Is this redundant, considering that we are using Ansible to deploy hosts?

Reply:
The dependencies are also used to run devel version on workstations,  run unit/integ tests and ensure that we are really deploying the required deps at runtime